app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js

```diff
@@ -18,20 +18,37 @@ const Migrations = ({
   createTransformationPlanRequestClick,
   isCreatingTransformationPlanRequest,
   finishedTransformationPlans,
-  redirectTo
+  redirectTo,
+  showConfirmModalAction,
+  hideConfirmModalAction,
+  fetchTransformationPlansAction,
+  fetchArchivedTransformationPlansUrl,
+  archivedTransformationPlans,
+  allArchivedPlanRequestsWithTasks
 }) => {
   const filterOptions = [
-    'Migration Plans Not Started',
-    'Migration Plans in Progress',
-    'Migration Plans Completed'
+    'Not Started Plans',
```

Comment:
ahh you are renaming them in an array. Can we also reference the strings as constants?